MidasCivil_Engine/Create/MetaData.cs

```diff
@@ -0,0 +1,110 @@
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using BH.oM.Adapters.MidasCivil;
+using BH.oM.Reflection.Attributes;
+
+
+namespace BH.Engine.Adapters.MidasCivil
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("An object containing various metadata for the model.")]
+        [Input("location", "Where the project is based.")]
+        [Input("description", "A short description of the project or the script that maybe useful to others.")]
+        [Input("discpline", "The discipline responsible for the model.")]
+        [Input("creationDate", "The creation date of the model inputted as yyyy-MM-dd. This will default to the current date if no date is provided.")]
+        [Input("reviews", "A list of reviews inputted as a review object.")]
+        [Output("An object containing various meta data for the model.")]
+        public static MetaData MetaData(string projectNumber = "", string projectName = "", string location = "", string client = "", 
+            string designStage = "", string projectLead = "", string revision = "", string author = "", object creationDate = null, string email = "", 
+            string description = "", string discipline = "", List<Review> reviews = null)
+        {
+            MetaData Data = new MetaData();
+            Data.ProjectNumber = projectNumber;
+            Data.ProjectName = projectName;
+            Data.Location = location;
+            Data.Client = client;
+            Data.DesignStage = designStage;
+            Data.ProjectLead = projectLead;
+            Data.Revision = revision;
+
+            Data.Author = author;
+            Data.Email = email;
+
+            Data.Description = description;
+            Data.Discipline = discipline;
+
+            Data.Reviews = reviews;
+
+            if (creationDate == null) { Data.CreationDate = DateTime.Now; }
+            else
+            {
+                Data.CreationDate = (DateTime)convertDate(creationDate);
+            }
+
+            return Data;
+        }
+
+        [Description("An object for creating a review to be inputted into metdata object.")]
+        [Input("reviewer", "A list of reviewers who have reviewed the model.")]
+        [Input("reviewDate", "The date when the model was reviewed by each reviewer either as a string in the format yyyy-MM-dd, or DateTime object.")]
+        [Input("approved", "The model is approved for it's intended use.")]
+        [Output("An object containing a review of the model.")]
+        public static Review Review(string reviewer = null, object reviewDate = null, List<string> comments = null, bool approved = false)
```

Comment:
This should sit in `Review.cs` under the Create folder - not in the same file as `MetaData.